plugins/modules/ndo_l3out_floating_svi_interface_path_attributes.py

```diff
@@ -0,0 +1,375 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Akini Ross (@akinross) <akinross@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: ndo_l3out_floating_svi_interface_path_attributes
+short_description: Manage L3Out Floating SVI Interface Path Attributes on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage L3Out Floating SVI Interface Path Attributes on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is only supported on ND v3.1 (NDO v4.3) and later.
+author:
+- Akini Ross (@akinross)
+options:
+  template:
+    description:
+    - The name of the template.
+    - The template must be an L3Out template.
+    - This parameter or O(template_id) is required.
+    type: str
+    aliases: [ l3out_template ]
+  template_id:
+    description:
+    - The ID of the L3Out template.
+    - This parameter or O(template) is required.
+    type: str
+    aliases: [ l3out_template_id ]
+  l3out:
+    description:
+    - The name of the L3Out.
+    - This parameter or O(l3out_uuid) is required.
+    type: str
+    aliases: [ l3out_name ]
+  l3out_uuid:
+    description:
+    - The UUID of the L3Out.
+    - This parameter or O(l3out) is required.
+    type: str
+  node_id:
+    description:
+    - The Anchor Node ID of the L3Out Floating SVI Interface.
+    type: str
+    required: true
+    aliases: [ node, anchor_node_id ]
+  encapsulation_type:
+    description:
+    - The encapsulation type of the interface.
+    type: str
+    required: true
```

Comment:
If we are making this always required I would default it to VLAN as this is the main use case.

Reply:
always thought we avoided defaults, but can agree with this one